ecosystem/typescript/sdk/src/aptos_types/authentication_key.ts

```diff
@@ -21,8 +21,12 @@ export class AuthenticationKey {
 
   static readonly ED25519_SCHEME: number = 0;
 
+  static readonly DERIVE_OBJECT_FROM_SEED_SCHEME: number = 254;
+
   static readonly DERIVE_RESOURCE_ACCOUNT_SCHEME: number = 255;
 
+  static readonly COLON: number = 58;
```

Comment:
Does TS have an equivalent of b"::" like Python?

Reply:
nope